packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -484,18 +458,11 @@ export interface IGarbageCollectorConfigs {
 	 */
 	readonly gcEnabled: boolean;
 	/**
-	 * Tracks if sweep phase is allowed for this document. This is specified during document creation and doesn't change
```

Comment:
Wait, we should still have a config for whether sweep is allowed for this document at all.  I'd want to see that in the logs to confirm.